src/components/ha-icon.ts

```diff
@@ -186,6 +186,7 @@ export class HaIcon extends LitElement {
 
   static styles = css`
     :host {
+      display: flex;
```

Comment:
This seems risky. In `ha-svg-icon` it's handled with a variable `display: var(--ha-icon-display, inline-flex)`. But I don't see a problem in the UI so this is theoretical.

Reply:
I would not do that during beta 😅